### PR TITLE
Edit readme + add simple function

### PR DESCRIPTION
**Pull request by contardog**
blabla

### CODE REVIEW

**Comment by Yoshinta** [comment]
I am going to review this

File: Contardo/template/simple_functions.py
Comment (Yoshinta) [inline Contardo/template/simple_functions.py:15]: this function only check if the input is even.
The command is to return True if the input is prime and False otherwise.
The input must be integer.

File: Contardo/template/buggy_function.py
Comment (Yoshinta) [inline Contardo/template/buggy_function.py:12]: I don't see any changes in the code. Only the notes.
Try ti check the angles->RA conversion.

File: Contardo/template/README.md
Comment (Yoshinta) [inline Contardo/template/README.md:13]: Good!
Maybe add examples.

**Review by Yoshinta — CHANGES_REQUESTED** [review]
Hi,
I wrote some suggestions in your code.

**Comment by bsipocz** [comment]
@contardog @Yoshinta - Thanks for opening the pull request and for the review(s)! Below I have provided a few comments about the *process* of the pull request and review. Note that we are not conducting a review of the code itself, because the exercise was about the overall process not the details of the code. Also, because there are pending suggestions, we will not merge this PR until the end of Astro Hack Week.

@contardog - it would have been good to provide a description for the PR, even if it duplicates a little the title.

@Yoshinta - you did not select "approve" or "request changes" when you submitted the review.  Without that a maintainer isn't sure if your review should be taken as a complete review or just general comments.

**Comment by bsipocz** [comment]
As a closure to AstroHackWeek, we go ahead and merge this now. Thank you both.

**Pull request merged**
